readme.md

```diff
@@ -36,7 +36,7 @@ MDX is a JSX in Markdown loader, parser, and renderer for ambitious projects. It
 ## Plugins
 
 - [inline-mdx](https://github.com/hamlim/inline-mdx.macro) - Babel [macro](https://github.com/kentcdodds/babel-plugin-macros) for transforming MDX inline.
-- [next-mdx](https://github.com/zeit/next-plugins/tree/master/packages/next-mdx) - MDX plugin for [Next.js](https://github.com/zeit/next.js).
+- [next-mdx](https://github.com/vercel/next.js/tree/canary/packages/next-mdx) - MDX plugin for [Next.js](https://github.com/vercel/next.js).
```

Comment:
rather than pointing to the canary release it may be good to point to the latest/HEAD version of code document
```suggestion
- [next-mdx](https://github.com/vercel/next.js/tree/HEAD/packages/next-mdx) - MDX plugin for [Next.js](https://github.com/vercel/next.js).
```